plugins/gamecenter/game_center.mm

```diff
@@ -226,6 +227,14 @@
 				hidden.push_back(description.hidden == YES);
 
 				replayable.push_back(description.replayable == YES);
+
+				NSNumber *number;
+				#ifdef __IPHONE_17_0
+				if (@available(iOS 17.0, *)) {
```

Comment:
We need both the macro and the availability checks, the former to allow building on pre-17.0 SDKs and the latter to silence compilation warnings when building on >17.0.

Reply:
Something like this will probably work without ifdefs (property should have default getter with the same name):
```obj-c++
if ([description respondsToSelector:@selector(rarityPercent)]) {
	number = [description performSelector:@selector(rarityPercent)];
}
```

Reply:
Much better, great suggestion! I confirmed this works with iOS 17.